### DISCUSSION TITLE
瀏覽器語言使用中文(繁體)登入後一片空白

### DISCUSSION THREAD

**Discussion by jasonwu1994** [FranzKafkaYu/x-ui#233] [Q&A] [answered]
問題重現: 將瀏覽器語言設定成中文(繁體)->移到最上面
chrome://settings/languages -> add Language ->Chinese (Traditional)->Move to top
這時候登入x-ui後將會是一片空白，如果是英文或中文(簡體)就正常
以上問題在0.3.3.15-0728及之後的版本都會發生

**Comment by KatsuraNTR** [comment]
終於知道爲什麽一片空白了

**Reply by jasonwu1994** [reply]
我弄好了幾天，以為版本有問題，看了0.3.3.15-0724(正常)到0728(異常)中間只改了README
就導致繁體介面進去一片空白

**Comment by FranzKafkaYu** [comment, marked as answer]
当前版本对于繁体的适配可能是有问题的，请尽量使用简体中文，当然也可选择英文。
后续将会看情况完善对繁体的支持。
